docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-page-dom.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: docs
+title: Examples of Working with Page DOM
+permalink: /documentation/test-api/selecting-page-elements/examples-of-working-with-page-dom.html
+---
+# Examples of Working with Page DOM
+
+This document shows how to work with page DOM in frequent real-world situations.
+
+* [Accessing Page Element Properties](#accessing-page-element-properties)
+* [Getting a Page Element Using Custom Logic](#getting-a-page-element-using-custom-logic)
+* [Checking if an Element is Available](#checking-if-an-element-is-available)
+* [Enumerating Elements Identified by a Selector](#enumerating-elements-identified-by-a-selector)
+
+## Accessing Page Element Properties
+
+To work with page elements, use TestCafe [selectors](selectors/README.md).
+Import the `Selector` function and call it passing a CSS selector inside.
```

Comment:
We can use not only CSS selector for `Selector` argument

Reply:
I see nothing wrong here.
At the first time, we create a selector in the easiest way possible. Because the main point here is getting selector properties. We don't worry much about how the selector is created.

In the next section, we show that we can create a selector not only with a CSS string, but also with a function.

Reply:
\cc @AlexanderMoskovkin  - maybe you see something wrong?

Reply:
It looks ok for me. We have the `Selector` topic where we describe all constructor options. Here we just show some particular cases of using

Reply:
ok, I don't mind